client/src/App.js

```diff
@@ -32,8 +33,12 @@ const App = () => {
 				<Route path="/admin/cohorts" element={<Cohorts />} />
 				<Route path="/admin/cohorts/addcohort" element={<CohortForm />} />
 				<Route path="/admin/cohorts/:id" element={<Cohort />} />
+				<Route path="/admin/cohorts/updatecohort/:id" element={<UpdateCohort />} />
 				<Route path="/admin/cohorts/:id/addtrainee" element={<AddTrainee />} />
-				<Route path="/admin/cohorts/:id/updatetrainee/:id" element={<UpdateTrainee />} />
+				<Route
+					path="/admin/cohorts/:id/updatetrainee/:id"
```

Comment:
Same as above for updatetrainee path="/admin/cohorts/:id/trainees/:id/edit" but I see this is not actually part of this PR so dont worry too much about it